ttnn/cpp/ttnn/operations/moreh/moreh_adam/moreh_adam_pybind.hpp

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "pybind11/cast.h"
+#include "pybind11/decorators.hpp"
+#include "ttnn/operation.hpp"
+#include "ttnn/operations/moreh/moreh_adam/moreh_adam.hpp"
+
+namespace py = pybind11;
+
+namespace ttnn::operations::moreh::moreh_adam {
+void bind_moreh_adam_operation(py::module& module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_adam,
+        "Moreh Adam Operation",
+        ttnn::pybind_overload_t{
```

Comment:
Overload is not needed if you are not overloading anything.
It is only required if your underlying function accepts a different set of arguments or arguments in binding have a different order. I don't see it the case here. In this case lines 22-67 can be removed completely.

Reply:
I’ve removed them as requested